doc/Glossary/index.md

```diff
@@ -6,74 +6,232 @@ nav: /:path/Glossary/
-| Transport channel                                       | A Kaa client component responsible for communication with a Kaa server. Refer to Transports for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
```

Comment:
There are multiple Access Tokens in this system. Endpoint Access Token, User Access token. Need to be more specific.